### ISSUE TITLE
md-list - not scrollable - ios

### ISSUE THREAD

**Issue by graphefruit** [angular/material#8313]
Hello Angular-Team,

I've found out, that on iOS 7 the md-list is not scrollable.

The expected behavior would be a smooth scrolling on iOS 7

**CodePen**: http://codepen.io/anon/pen/bpQmOR

**Video**: I've made a small video to demonstrate it https://vid.me/YetS

**Found**:
I've found out what is causing this issue, if `md-list-item` has a `ng-click`, the content cant be scrolled.

If I *remove* `ng-click` the scrolling is smooth.

I've also tried to wrap the md-list-item inside with an md-button (still not scroll able).
So the problem seems to appear with the nested DOM which is generated.

```
<div class="md-button _md-no-style ">
    <button class="_md-no-style md-button md-ink-ripple" type="button" ng-transclude="" aria-label="0"
            ng-click="null"></button>
    <div class="_md-list-item-inner ng-binding">
        0
    </div>
    <div class="flex"></div>
    <div class="_md-secondary-container"></div>
</div>
```
**Angular Versions**:
-  Angular Version: 1.5.5
-  Angular Material Version: 1.1.0 - RC 4

**Additional Information**:
-  Browser Type: Safari
-  Browser Version: Safari 7
-  OS: iOS 7

Thanks in Advance
Graphefruit

Update:
Found that if I set the button possition to relative the scrolling is smooth again.
`
.md-button{
  position:relative!important;
}
`
The problem is, that the content will be on the right and the button which is clickable is on the left. (UI destroyed)
![image](https://cloud.githubusercontent.com/assets/6260244/14978245/bc81b8b4-111a-11e6-816a-60125cf6e199.png)
So it seems like the button is making some weird things while try to scroll.

**Comment by DevVersion** [comment]
@graphefruit just investigated at the issue on his real device again, and we were able to solve the issue by using the `skipClickHijack` config method.

Will investigate at the `clickHijack` code now.

**Comment by DevVersion** [comment]
Closing this now, since this will be fixed with #9362
